### PR TITLE
Action: Remove deprecated type attribute in ir.ui.view + Cleanup and Refactor

### PR DESCRIPTION
**Pull request by bwrsandman**
This is a simple cleanup of this module due while fixing a deprecation
- Remove deprecated `type` attribute in ir.ui.view
-  Refactor module:
  - Separate data, demo from views
  - Separate view, data and python scripts by model
  - Use CamelCase for class names
- Fixup issues with MgmtsystemAction
  - Remove accidental attempt to write to context column
- Update translation files

The changes are in b5d5e7f, 957db71 and b954002
There is only refactoring in 8926c62
And 6c28833 is only pot/po files

### CODE REVIEW

File: mgmtsystem_action/models/mgmtsystem_action.py [hunk 1]
```diff
@@ -57,11 +58,9 @@ class mgmtsystem_action(orm.Model):
     }
 
     def create(self, cr, uid, vals, context=None):
-        vals.update({
-            'reference': (
-                self.pool.get('ir.sequence').get(cr, uid, 'mgmtsystem.action'))
-        }, context=context)
-        return super(mgmtsystem_action, self).create(
+        sequence_pool = self.pool['ir.sequence']
+        vals.update(reference=sequence_pool .get(cr, uid, 'mgmtsystem.action'))
```
Comment (lmignon) [inline mgmtsystem_action/models/mgmtsystem_action.py]: remove space between `sequence_pool` and `get` in `sequence_pool .get(cr, uid, 'mgmtsystem.action')`

**Comment by lmignon** [comment]
Once the typo will be fixed :+1:

**Comment by max3903** [comment]
:+1:

**Comment by sbidoul** [comment]
:+1: (code review)

**Comment by bwrsandman** [comment]
Thank you, could you also review #59. These are the same kind of changes.

**Pull request merged**